src/app/georeferencer/qgsgeorefdatapoint.cpp

```diff
@@ -22,18 +22,17 @@
 #include "qgsgeorefdatapoint.h"
 
 QgsGeorefDataPoint::QgsGeorefDataPoint( QgsMapCanvas *srcCanvas, QgsMapCanvas *dstCanvas,
-                                        const QgsPointXY &pixelCoords, const QgsPointXY &mapCoords,
-                                        const QgsCoordinateReferenceSystem proj, bool enable )
+                                        const QgsPointXY &sourceCoordinates, const QgsPointXY &destinationMapCoords,
+                                        const QgsCoordinateReferenceSystem &destinationCrs, bool enable )
```

Comment:
I'm hesitant about the name `destinationCrs` , I understand this is the crs of the 'destination' of this point, but it can be confused with the destination crs of the raster.  Could the name simply be `pointCrs` as it belongs to the point and are a property of it, rather than something that should be changed.